scenes/quests/story_quests/template/0_intro/intro.dialogue

```diff
@@ -4,9 +4,10 @@
 ~ start
 do animation_player.play(&"walk_on")
 do animation_player.animation_finished
-Change this text by selecting the Cinematic node and opening the Dialogue resource in the Inspector.
+Inside your StoryQuest intro folder, double click intro.dialogue to edit the text.
```

Comment:
I was about to tell you about using rich text for setting mono font in the file name like in:
```
Inside your StoryQuest intro folder, double click [code]intro.dialogue[/code] to edit the text.
```
But it doesn't work because of our current font! So I said nothing.

By the way, I really think that it's better to access the dialogue by opening the intro scene, selecting the Cinematic node, then opening the Dialogue from the Inspector (what it was previously mentioned).

Reply:
I guess the thinking behind this change is that it's harder to edit the wrong dialogue file if you go via the filesystem.

The other approach would be to swap the instruction round:

1. Drag the correct dialogue (your copy, not the template) onto the cinematic node
2. Now it doesn't matter whether you go via the inspector or the filesystem